DOWNLOAD_STATS.md

```diff
@@ -5,14 +5,30 @@
+| `@nolyfill/array.prototype.copywithin` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.copywithin.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.copywithin) |
+| `@nolyfill/array.prototype.entries` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.entries.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.entries) |
+| `@nolyfill/array.prototype.every` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.every.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.every) |
 | `@nolyfill/array.prototype.find` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.find.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.find) |
 | `@nolyfill/array.prototype.findlast` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.findlast.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.findlast) |
 | `@nolyfill/array.prototype.findlastindex` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.findlastindex.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.findlastindex) |
 | `@nolyfill/array.prototype.flat` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.flat.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.flat) |
 | `@nolyfill/array.prototype.flatmap` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.flatmap.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.flatmap) |
+| `@nolyfill/array.prototype.flatmap` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.flatmap.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.flatmap) |
+| `@nolyfill/array.prototype.foreach` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.foreach.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.foreach) |
+| `@nolyfill/array.prototype.join` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.join.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.join) |
```

Comment:
Nobody uses `array.prototype.join` on npm. All of its dependents are from ljharb.